src/backend/vulkan/src/lib.rs

```diff
@@ -321,13 +355,33 @@ impl hal::Instance<Backend> for Instance {
                 .unwrap_or(std::ptr::null_mut())
         });
 
+        // Pick the latest version of Vulkan available.
+        let api_version = match entry.try_enumerate_instance_version() {
+            // Vulkan 1.1+
+            Ok(Some(version)) => Version(version),
+
+            // Vulkan 1.0
+            Ok(None) => Version::V1_0,
+
+            // Ignore out of memory since it's unlikely to happen and `Instance::create` doesn't have a way to express it in the return value.
+            Err(err) if err == vk::Result::ERROR_OUT_OF_HOST_MEMORY => {
```

Comment:
panicking here doesn't sounds like "ignore" :)
Should we just return `hal::UnsupportedBackend` instead on error? We can log the actual error in `warn!`